pkg/webui/console/store/middleware/logics/lib.js

```diff
@@ -89,7 +94,20 @@ const createRequestLogic = function(
         } else {
           // Otherwise, dispatch the fail action and report it to Sentry.
           if (isUnknown(e) || isInvalidArgumentError(e)) {
-            Sentry.captureException(failAction(e))
+            Sentry.withScope(scope => {
+              scope.setExtras(e)
+              if (e.message) {
+                scope.setFingerprint(getBackendErrorId(e))
+                Sentry.captureException(new Error(getBackendErrorId(e)))
+              } else if (e.payload && e.payload.message) {
```

Comment:
When does that happen that a fail action is thrown? That seems odd to me, the `process()` function should dispatch API actions that should fail with backend error objects or request errors. Only the request logic itself here is wrapping the error into the fail action.

Reply:
sentry.io/organizations/the-things-industries/issues/1739001198/?project=2682566&query=is%3Aunresolved  check this

Reply:
https://sentry.io/organizations/the-things-industries/issues/1739001198/events/024287ed737f4e8590651102b6c97653/?project=2682566

Why `isBackend` is false for this error? It has all the attributes that `isBackend` checks, but it still goes into this condition `isUnknown || isInvalidArgumentError`. This and many other errors mean that we cant just ignore `isUnknown`, but the problem is that the request errors and unrecognized backend errors are mixed there, meaning Sentry will have a hard time distinguishing between them.

Reply:
So what's the conclusion? Can you check where we wrongfully throw the whole action instead of just the error payload? We need an issue to fix that.

Reply:
Yes we did it here, I wrapped the exception with `failAction` and reported it like that to Sentry. That is fixed now.